mitodl/mit-open
# Shanbady/additional details on offeror channel pages

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4327

### Description (What does it do?)
This PR adds an info box to offerer pages (/c/offeror/mitpe)

[figma](https://www.figma.com/proto/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?page-id=107%3A2479&node-id=3774-68962&viewport=182%2C371%2C0.5&t=SgD1ouufEbDt17yz-1&scaling=scale-down&starting-point-node-id=2197%3A31345&show-proto-sidebar=1)

### Screenshots (if appropriate):
<img width="1334" alt="Screenshot 2024-06-03 at 1 26 54 PM" src="https://github.com/mitodl/mit-open/assets/196425/4060fd14-beef-4f57-b9e6-fc4a1c53a156">
<img width="1365" alt="Screenshot 2024-06-03 at 1 28 29 PM" src="https://github.com/mitodl/mit-open/assets/196425/f0226956-4a19-4497-95e7-3c9f202580f6">


### How can this be tested?
1. Checkout this branch
2. make sure you have offerers loaded otherwise run ```python manage.py update_offered_by```
3. go to the different offerer pages http://od.odl.local:8063/c/offeror/xpro http://od.odl.local:8063/c/offeror/mitpe http://od.odl.local:8063/c/offeror/xpro http://od.odl.local:8063/c/offeror/mitx and see that the info box displays the appropriate data for each
